sections/07_conclusions.md

```diff
@@ -1,15 +1,60 @@
 ## Conclusions
 
-Final thoughts and future outlook here. The Discussion will give an overview
-and the Conclusion will provide a short, punchy take home message.
+Deep learning-based methods now represent the state of the art in a diverse
+array of tasks in patient and disease categorization, fundamental biological
+study, genomics, and treatment development.  We return to our central question:
+given this rapid progress, has deep learning transformed the study of human
+disease?  Though the answer is highly dependent on the specific domain and
+problem being addressed, we conclude that deep learning has not *yet* realized
+its transformative potential or induced a strategic inflection point.  Despite
+its dominance over competing machine learning approaches in many of the areas
+reviewed here and quantitative improvements in predictive performance, deep
+learning has not yet qualitatively "solved" those problems that were previously
+"unsolved".
 
-Points to mention based on discussion thus far that may make the bar for
-conclusions:
+As an analogy, consider recent progress in conversational speech recognition.
+Since 2009 there have been drastic performance improvements, with error rates
+dropping from more than 20% to less than 6% [@tag:Speech_recognition] and
+finally approaching or exceeding human performance in the past year
+[@arxiv:1610.05256 @arxiv:1703.02136] `TODO: working on a second source for this
+error trajectory from a talk by Eric Horvitz`. The phenomenal  improvements on
+benchmark datasets are undeniable, but the successes of the early 2010s did not
```

Comment:
Good point, this is unclear.  I'm thinking of the left image from https://twitter.com/amram/status/845748240033050624  However, I can't directly use it because I don't know the source, and Eric is likely too busy being the new head of MSR to reply.  http://www.businessinsider.com/ibm-edges-closer-to-human-speech-recognition-2017-3 shows comparable numbers but with less granularity.

I changed this to "...are undeniable, but halving the error rates on these benchmarks did not fundamentally transform...".  Still not perfect, but that's my intended message.

No one is pushing back on my conversational speech example, so I removed: `TODO: this is debatable, maybe need a different example or to clarify what is meant by "conversational" speech`